README.md

```diff
@@ -14,12 +14,16 @@ To use this plugin, add the dependency to your project's build.gradle file:
 
 ```groovy
 buildscript {
-    repositories {
-        jcenter()
-    }
-    dependencies {
-        classpath 'com.apollographql.apollo:apollo-gradle-plugin:x.y.z'
-    }
+  repositories {
+    jcenter()
+  }
+  dependencies {
+    classpath 'com.apollographql.apollo:apollo-gradle-plugin:x.y.z'
+  }
+}
+
+dependencies {
+  compile 'com.apollographql.apollo:apollo-runtime:x.y.z'
```

Comment:
Added documentation for the `dependencies` block. I left using `compile` configuration. AGP already warns users to use `implementation` for 3.0.0+.